### DISCUSSION TITLE
Migrating custom layout from pimcore 6 to 11

### DISCUSSION THREAD

**Discussion by anshu1thakur** [pimcore/pimcore#17612] [Q&A / Support]
I am migrating my classes from Pimcore 6 to Pimcore 11. However, I encountered a problem with migrating numerous custom layouts as well. If anyone knows an easy way or trick to migrate them quickly, it would be greatly appreciated.

**Comment by EliShklovsky** [comment]
Hi @anshu1thakur 
You should open the exported json in text editor and make some search/replace:

childs 
children

AppBundle\\
App\\

\\AppBundle\\
\\App\\

"maxItems": "",
"maxItems": null,

"rows": "",
"rows": null,

 "cols": "",
 "cols": null,

Erase completely:
 "fieldtype": "button",

Hope that helps
Regards,
Eli
